client/menus/main.lua

```diff
@@ -128,10 +128,6 @@ end
 menu.onClose = function()
     inMenu = false
     stopDragCam()
-    lib.showTextUI('Press [E] to tune your car', {

```

Comment:
Why is this removed?

Reply:
it was removed because when an admin finishes modifying their car it shows up and can not be removed unless you go to  the mechanic shop. Simple fix was to just remove it because players can still press E to access the shop though the text isn't showing and admins don't have the text on their screen.